app/src/main/java/net/gsantner/markor/frontend/filesearch/FileSearchResultSelectorDialog.java

```diff
@@ -217,11 +226,15 @@ public View getGroupView(final int groupPosition, final boolean isExpanded, fina
             }
             textView.setText(groupInfo.toString());
 
-            final int iconResId = groupInfo.isDirectory || groupInfo.children.isEmpty() ? 0 : isExpanded
-                    ? R.drawable.ic_baseline_keyboard_arrow_up_24
-                    : R.drawable.ic_baseline_keyboard_arrow_down_24;
+            final int iconResId;
+            if (groupInfo.isDirectory || groupInfo.children.isEmpty()) {
+                iconResId = 0;
+            } else if (isExpanded) {
+                iconResId = R.drawable.ic_baseline_keyboard_arrow_up_24;
+            } else {
+                iconResId = R.drawable.ic_baseline_keyboard_arrow_down_24;
```

Comment:
Right now we don't specify an icon for file name results. Would it be better to use a folder or file icon instead?

I am adding a trailing slash to folders in order to indicate that they are folders

Reply:
Usability improvements are sure not wrong. There is also the bug with the bottom snackbar not dismissing, or continue searching wrongly.